ci/test-stable-perf.sh

```diff
@@ -30,6 +30,13 @@ for test in tests/*.rs; do
   _ cargo test --verbose --jobs=1 --features="$FEATURES" --test="$test"
 done
 
+echo --- "( cd programs/native/bpf_loader ; cargo test --verbose )"
```

Comment:
Running these in ci/test-stable.sh should be sufficient.   These programs have no cuda, erasure, chacha in them